purchase_exception/wizard/purchase_exception_confirm.py

```diff
@@ -15,5 +15,7 @@ class PurchaseExceptionConfirm(models.TransientModel):
     def action_confirm(self):
         self.ensure_one()
         if self.ignore:
+            self.related_model_id.button_draft()
```

Comment:
Why you execute button_draft before the confirm ?

Reply:
Because button_confirm() only works from status 'draft', and you still want the logic https://github.com/odoo/odoo/blob/11.0/addons/purchase/models/purchase.py#L346 to be executed.